CIP-0000.md

```diff
@@ -0,0 +1,198 @@
+
+It is highly recommended that a single CIP contain a single key proposal or new idea.
+
+A CIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement as to be determined by the CIP editors during the PR review process. For more information on the CIP Editor role, see the appropriate section below.
+
+
+# CIP Workflow
+## Shepherding a CIP
+Parties involved in the process are you, the champion or _CIP author_, the CIP editors, and the Celo Core Developers (core developers are the developers working on the Celo client).
+
+Before you begin writing a formal CIP, you should vet your idea. Ask the Celo community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is thus recommended to open a discussion thread on Issues section of the CIP process to do this, but you can also use the Celo Discord and the [Celo Forum](https://forum.celo.org) and engage with the community about your idea.
+
+In addition to making sure your idea is original, it will be your role as the author to make your idea clear to reviewers and interested parties, as well as inviting editors, developers and community to give feedback on the aforementioned channels. You should try and gauge whether the interest in your CIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the work required to implement a Core CIP will be much greater than for a CRC and the CIP will need to be reviewed carefully by the Celo community. Negative community feedback will be taken into consideration and may prevent your CIP from moving past the Draft stage.
+
+Building consensus with the Celo community must adhere to the [Celo Code of Conduct](https://celo.org/code-of-conduct) and the [Celo Community Tenets](https://celo.org/community). Your CIP must also be in-line with Celo’s Community Tenets. For example, proposals like changing consensus mechanisms to Proof-of-Work doesn’t follow Celo’s vision because it won’t make Celo accessible to everyone with a smartphone and will therefore not help advance Celo.
+
+## Core CIPs
+For Core CIPs, given that they require Celo’s client implementation to be considered **Final** (see “CIPs Process” below), the blockchain author will need to either provide an implementation for the celo-blockchain client or convince clients to implement your CIP.
+
+The best way to get client implementers to review your CIP is to present it on a Celo AllCoreDevs call. You can request to do so by posting a comment linking your CIP on an agenda issue for an upcoming call on the Github repository.
```

Comment:
I feel we should encourage an async method here- e.g. by posting it in the `#governance` channel on Discord and on the Forum (we should also probably choose which one of those is primary!)

Reply:
I think, in reference to the organizing what to include on ACD calls, we probably should stick to Github. Each call can have it's own issue link and people can comment on what they want included in the call. If breakout sessions are needed post calls to discuss things more, new issue tickets can be created per breakout session.

Discord would make it super hard to track things for a specific topic as things are more organic. Forum can function the same way as a Github issue ticket as a discussion space for a proposed CIP.

Reply:
I'm happy with any of the above, but we'll just need to pick one!